### PR TITLE
feature/SM-3393: replace job-1 with generate-sample-inspection

### PR DESCRIPTION
**Pull request by ccurran01**
## Description

replace job-1 services with generate sample inspection services

add street manager data as a dependency
 - as message interface definitions are shared between api and worker
 these have been moved from the worker into street manager data

## Checklist

- [ ] All unit tests passing
- [ ] Code coverage suitable
- [ ] Branch named {feature|hotfix|task}/{SM-.*}
- [ ] If your pull request depends on any other, please link them
- [ ] Changes approved by your team
- [ ] Changes approved by another team
- [ ] Commit messages are meaningful
- [ ] Target is green https://circleci.com/gh/departmentfortransport/street-manager-worker/tree/master
- [ ] Int UI tests passing https://circleci.com/gh/departmentfortransport/street-manager-int/tree/master
- [ ] Add `DO NOT MERGE` if you want to postpone merge

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -11,15 +11,12 @@ export const SQS_CONNECT_TIMEOUT_MS = process.env.SQS_CONNECT_TIMEOUT_MS || (5 *
...
-export const JOB_1_PGPASSWORD = process.env.JOB_1_PGPASSWORD || 'app'
-export const JOB_1_PGMINPOOLSIZE = process.env.JOB_1_PGMINPOOLSIZE || '5'
-export const JOB_1_PGMAXPOOLSIZE = process.env.JOB_1_PGMAXPOOLSIZE || '10'
-export const JOB_1_PGSSL = process.env.JOB_1_PGSSL === 'true' || false
+export const GENERATE_SAMPLE_INSPECTION_MAX_JOBS = process.env.GENERATE_SAMPLE_INSPECTION_MAX_JOBS || '10'
+export const GENERATE_SAMPLE_INSPECTION_PGHOST = process.env.GENERATE_SAMPLE_INSPECTION_PGHOST || 'localhost'
+export const GENERATE_SAMPLE_INSPECTION_PGPORT = process.env.GENERATE_SAMPLE_INSPECTION_PGPORT || '5432'
+export const GENERATE_SAMPLE_INSPECTION_PGDATABASE = process.env.GENERATE_SAMPLE_INSPECTION_PGDATABASE || 'work'
+export const GENERATE_SAMPLE_INSPECTION_PGUSER = process.env.GENERATE_SAMPLE_INSPECTION_PGUSER || 'app'
+export const GENERATE_SAMPLE_INSPECTION_PGPASSWORD = process.env.GENERATE_SAMPLE_INSPECTION_PGPASSWORD || 'app'
```
Comment (cpaul92) [inline src/config.ts:19]: Can we remove these defaults since this is a public repo

Comment (ccurran01) [reply src/config.ts:19]: 👍

Comment (ccurran01) [reply src/config.ts]: Removed

**Comment by ccurran01** [comment]
> Do you need/have an ops repo update for master-docker-compose too to provide the environment variables you added/changed?

Have one for adding the adding the new url to the api but not for renaming job_1 so will probably need to do that

**Pull request merged**